block4_debug/README.md

```diff
@@ -0,0 +1,120 @@
+# Debuggning
```

Comment:
@SGenheden Häftigt! Mergar och kollar mer på kodboken dold sida. Vill se det i sin helhet där, enklare.

Reply:
@SGenheden Superfint! 
Här finns länkar till dolda sidor på Kodboken - om du eller någon annan vill reviewa?

Debug Race: https://www.kodboken.se/start/skapa-spel/uppgifter-i-scratch/debugrace?chpt=0
 
DeBug: https://www.kodboken.se/start/skapa-spel/uppgifter-i-scratch/debug 

Önskvärt: Bilder in på de olika DeBug-uppgifterna vore najs, om de skiljer sig från den första bilden på nåt sätt så det blir piffigt vill säga.. .